src/main.rs

```diff
@@ -211,6 +211,15 @@ fn _main() -> PaziResult {
         }
 
         if flags.is_present("interactive") {
+            // Capture ctrl-c so calling script
```

Comment:
I think we want to do this regardless of interactive or not; ctrl-c is still a thing for e.g. `z foo` even if the window is tiny.